Upgrading.md

```diff
@@ -0,0 +1,175 @@
+> [remove the Clear Containers packages](#remove-clear-containers-packages) since doing so will
+> *delete the default Clear Containers configuration file from your system*.
+
+## Remove Clear Containers packages
+
+> **Warning**: If you have modified your
+> [Clear Containers configuration](https://github.com/clearcontainers/runtime#configuration),
+> you may wish to make a safe copy of the configuration file before removing the
+> packages since doing so will *delete the default configuration file*
+
+### Fedora
+
+```
+$ sudo -E dnf remove cc-runtime\* cc-proxy\* cc-shim\* linux-container clear-containers-image qemu-lite
+$ sudo rm /etc/yum.repos.d/home:clearcontainers:clear-containers-3.repo
+```
+
+### Ubuntu
+
+```
+$ sudo apt-get purge cc-runtime\* cc-proxy\* cc-shim\* linux-container clear-containers-image qemu-lite
+$ sudo rm /etc/apt/sources.list.d/clear-containers.list
+```
+
+## Disable old container manager configuration
+
+Assuming a Docker installation, remove the docker configuration for Clear
+Containers:
+
+```
+$ sudo rm /etc/systemd/system/docker.service.d/clear-containers.conf
+```
+
+## Install Kata Containers
+
+Follow one of the [installation guides](https://github.com/kata-containers/documentation/tree/master/install).
+
+## Create a Kata Container
+
+```
+$ sudo docker run -ti busybox sh
+```
+
+# Upgrade Kata Containers
+
+As shown in the
+[installation instructions](https://github.com/kata-containers/documentation/blob/master/install),
+Kata Containers provide binaries for popular distributions in their native
+packaging formats. This allows Kata Containers to be upgraded using the
+standard package management tools for your distribution.
+
+# Appendices
+
+## Assets
+
+Kata Containers requires additional resources to create a virtual machine
+container. These resources are called
+[Kata Containers assets](https://github.com/kata-containers/documentation/blob/master/architecture.md#assets)
+which comprise a guest kernel and a root filesystem or initrd image. This
+section describes when these components are updated.
+
+Since the official assets are packaged, they are automatically upgraded when
+new package versions are published.
+
+> **Warning**: Note that if you are using custom assets (by modifying the
+> [Kata Runtime configuration > file](https://github.com/kata-containers/runtime/#configuration)),
+> it is your responsibility to ensure they are updated as necessary.
+
+### Guest kernel
+
+The `kata-linux-container` package contains a Linux\* kernel based on the
+latest vanilla version of the
+[longterm kernel](https://www.kernel.org/)
+and including a small number of
+[patches](https://github.com/kata-containers/packaging/tree/master/kernel).
+
+The `Longterm` branch is only updated with
+[important bug fixes](https://www.kernel.org/category/releases.html)
+meaning this package will only be updated when necessary.
+
+The guest kernel package will be updated when a new longerm kernel is released
```

Comment:
Rewrite + missing "t" in "longterm": The guest kernel package is updated when a new longterm kernel is released